### PR TITLE
improved 0.9.3 support

### PR DESCRIPTION
**Pull request by NITEMAN**
Add support for wal-enable-logging parameter.
Only add wal-enable-logging and wal-dir to config file if version > 0.9.3

### CODE REVIEW

File: templates/influxdb.conf.erb [hunk 1]
```diff
@@ -26,9 +26,13 @@ hostname = "<%= @hostname -%>"
 [data]
   dir = "<%= @data_dir -%>"
   max-wal-size = <%= @max_wal_size %>
-  wal-dir = "<%= @wal_dir -%>"
   wal-flush-interval = "<%= @wal_flush_interval -%>"
   wal-partition-flush-delay = "<%= @wal_partition_flush_delay -%>"
+<%- if scope.function_versioncmp([@version, '0.9.3']) >= 0 -%>
```
Comment (n1tr0g) [inline templates/influxdb.conf.erb]: cool. I didn't know I can call puppet functions directly from the template !!! tnx

**Comment by n1tr0g** [comment]
Hey NIETMAN thank you for this PR, however I already updated the module with the 0.9.3 changes the other week, but forgot to commit. 

Also I added some other parameters which are missing here. Sorry for the trouble :(

File: templates/influxdb.conf.erb (hunk 1 above)
Comment (NITEMAN) [inline templates/influxdb.conf.erb]: Straight from puppetlabs/apache ;)

**Comment by NITEMAN** [comment]
I'ts my fault @n1tr0g , I had this 3 weeks ago but not commited ;)

**Pull request closed without merging**